### DISCUSSION TITLE
HalfWidth not working(?)

### DISCUSSION THREAD

**Discussion by axelcasauran** [serenity-is/Serenity#6272] [Help]
Hi,

I tried using HalfWidth but the design looks not half. There is a huge space in between the controls. Any thoughts?

![2022-02-15 21_14_58-Window](https://user-images.githubusercontent.com/13443821/154069532-1608ecc2-1041-4b43-9be5-ce0911fc38cd.png)

**Comment by wezmag** [comment]
The generated html added bootstrap's ".col-md-6" class to the field div.
That class makes the div half width.
``` css
@media (min-width: 768px)
.col-sm-6 {
    width: 50%;
}
```

To fix your problem, you will need to change the caption width to a smaller value.
Find the generated LESS file, the file name should be `site.somemodulename.less` in the `wwwroot/Content/site` folder.
Inside the file, you should see something like below.
``` LESS
.s-SomeModuleName-SomeRowDialog {
    > .size { width: 450px; }
    .caption { width: 120px; }
}
```
and then change the `.caption` `width` to a smaller value.

**Reply by axelcasauran** [reply]
I use **LabelWidth**(#) property and it works.
